CoreTesting/src/au/gov/asd/tac/constellation/testing/TestNotificationsAction.java

```diff
@@ -64,14 +64,8 @@ public TestNotificationsAction(final GraphNode context) {
     public void actionPerformed(final ActionEvent e) {
         // plugin notifications
         final Plugin plugin = new TestNotificationsPlugin();
-        final PluginParameters pp = plugin.createParameters();
-
-        final PluginParametersSwingDialog dialog = new PluginParametersSwingDialog(Bundle.CTL_TestNotificationsAction(), pp);
-        dialog.showAndWait();
-        if (PluginParametersDialog.OK.equals(dialog.getResult())) {
-            PluginExecution.withPlugin(plugin).withParameters(pp).executeLater(context.getGraph());
-        }
```

Comment:
With the removal of this code, there are now some unused imports that should also be removed.

Reply:
Done.